src/tactic/ui/tools/pipeline_wdg.py

```diff
@@ -7477,14 +7543,14 @@ def add_item_behaviors(self, el):
                         search_type: searchType,
                         project_code: projectCode,
                     }
-
                     server.p_execute_cmd(document_cmd, document_kwargs)
                     .then(function(ret_val){
                         top.removeClass("spt_unsaved_item");
                         var on_complete = function() {
                             var refreshedRow = spt.table.get_row_by_search_key(search_key);
                             refreshedRow.setAttribute("spt_group_level", 2);
-                            spt.table.select_row(refreshedRow);
+                            var documentItem = refreshedRow.getElement(".spt_document_item");
```

Comment:
Can you revert this? I made this change since spt.table.select_row is the better way to select a row.